conda-forge.yml

```diff
@@ -1,14 +1,9 @@
 build_platform:
-  linux_aarch64: linux_64
-  linux_ppc64le: linux_64
   osx_arm64: osx_64
   linux_ppc64le: linux_64
   linux_aarch64: linux_64
```

Comment:
@setu4993 
The section for `build_platform` contained duplicate key-value pairs. The bot doesn't know how to deal with that, because it breaks the assumption that it's +/- a dictionary.

Reply:
Got it, I hadn't even noticed it. I just assumed that file was auto-generated and haven't needed to check it anytime recently.